columnize.go

```diff
@@ -65,18 +67,22 @@ func getWidthsFromLines(config *Config, lines []string) []int {
 // returns a sprintf-style format string which can be used to print output
 // aligned properly with other lines using the same widths set.
 func (c *Config) getStringFormat(widths []int, columns int) string {
+	// Create the buffer
+	// 8 bytes per character * 8 characters per column * ~num columns
+	buf := bytes.NewBuffer(make([]byte, 0, 8*len(widths)*8))
+
 	// Start with the prefix, if any was given.
-	stringfmt := c.Prefix
+	buf.WriteString(c.Prefix)
```

Comment:
Bummer that we are swallowing an error here. Is the buffer wrapper necessary or can we just make a `[]byte` and use `append()`? Maybe at least do a `_, _ = buf.Write(c.Prefix)` to make it clear that we are intentionally throwing it out.